### PR TITLE
feat(swag): add Jina swag opportunity

### PR DESCRIPTION
**Pull request by Nishil07**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Added details of Jina AI swag opportunity

Fixes https://github.com/swapagarwal/swag-for-dev/issues/832

### CODE REVIEW

**Comment by aslafy-z** [comment]
Thank you @Nishil07 for your contribution, please rebase your branch. I fixed the CI here: https://github.com/swapagarwal/swag-for-dev/pull/835

**Comment by Nishil07** [comment]
@aslafy-z Have rebased please review it. Thanks!

File: data.json [hunk 1]
```diff
@@ -359,6 +359,15 @@
     "dateAdded": "2020-09-18T21:33:20.000Z",
     "tags": ["software", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Jina AI",
+    "difficulty": "medium",
+    "description": "Love Jina, get swag! Make a PR, write a blog post, or provide awesome tech support and you'll get a notepad, stickers, keyring, pen and tote bag",
+    "reference": "https://jina.ai/2021/07/12/swag.html",
```
Comment (aslafy-z) [inline data.json:366]:
```suggestion
    "reference": "https://jina.ai/blog/swag/",
```

File: data.json [hunk 2]
```diff
@@ -359,6 +359,15 @@
...
+    "image": "https://jina.ai/assets/images/blog/swag/advocate_swag.jpg",
+    "dateAdded": "2021-07-23T16:38:20.000Z",
+    "tags": ["notepad", "stickers", "keyring", "pen", "tote bag"]
```
Comment (aslafy-z) [inline data.json:369]:
```suggestion
    "tags": ["accesories", "stickers"]
```

File: data.json [hunk 3]
```diff
@@ -359,6 +359,15 @@
     "dateAdded": "2020-09-18T21:33:20.000Z",
     "tags": ["software", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Jina AI",
+    "difficulty": "medium",
+    "description": "Love Jina, get swag! Make a PR, write a blog post, or provide awesome tech support and you'll get a notepad, stickers, keyring, pen and tote bag",
```
Comment (aslafy-z) [inline data.json:365]:
```suggestion
    "description": "Love Jina, get swag! Make a PR, write a blog post, or provide awesome tech support and you'll get a notepad, stickers, keyring, pen and tote bag.",
```

**Comment by aslafy-z** [comment]
Awesome, thanks! I updated the link and the tags. Once you applied the reviews I'll merge your PR!

File: data.json [hunk 4]
```diff
@@ -359,6 +359,15 @@
     "dateAdded": "2020-09-18T21:33:20.000Z",
     "tags": ["software", "hacktoberfest", "expired"]
   },
+  {
+    "name": "Jina AI",
+    "difficulty": "medium",
+    "description": "Love Jina, get swag! Make a PR, write a blog post, or provide awesome tech support and you'll get a notepad, stickers, keyring, pen and tote bag.",
```
Comment (aslafy-z) [inline data.json:365]:
```suggestion
    "description": "Love Jina, get swag! Make a PR, write a blog post, or provide awesome tech support and you'll get a notepad, stickers, keyring, pen and tote bag. Star the repository and join online communities to get some stickers and fridge magnets.",
```

File: data.json [hunk 5]
```diff
@@ -359,6 +359,15 @@
...
+    "reference": "https://jina.ai/blog/swag/",
+    "image": "https://jina.ai/assets/images/blog/swag/advocate_swag.jpg",
+    "dateAdded": "2021-07-23T16:38:20.000Z",
+    "tags": ["accessories", "stickers"]
```
Comment (aslafy-z) [inline data.json:369]:
```suggestion
    "tags": ["accessories", "stickers", "accessories"]
```

**Comment by aslafy-z** [comment]
Thank you @Nishil07!

**Pull request merged**